src/cloudai/schema/test_template/jax_toolbox/slurm_command_gen_strategy.py

```diff
@@ -262,11 +263,11 @@ def set_xla_flags(test_name: str, profile_enabled: bool):
 
         # Combine both parts into the run script content
         run_script_content = profile_content + perf_content
-        run_script_path = os.path.join(cmd_args["output_path"], "run.sh")
+        run_script_path = Path(cmd_args["output_path"]) / "run.sh"
         with open(run_script_path, "w") as run_file:
             run_file.write("\n".join(run_script_content))
         os.chmod(run_script_path, 0o755)
-        return run_script_path
```

Comment:
Do you think it make sense to return `Path` here too?

Reply:
Actually, we don't need to return anything since the output value is not used.

Reply:
Even better :)